back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/utils.rb

```diff
@@ -264,6 +266,17 @@ def user_locales(serialized_models)
           users.pluck('locale').uniq.sort
         end
 
+        def parse_yml(content)
+          YAML.safe_load(content, aliases: true, permitted_classes: [Date, Symbol, Time]) # rubocop:disable Security/YAMLLoad
```

Comment:
will remove the outdated directive

Reply:
I'm not sure I got it. `safe_load` is not outdated, is it? 

Also, why do you need `# rubocop:disable Security/YAMLLoad`? `safe_load` doesn't have any security issues.

Also, [here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Security/YAMLLoad) they say that in Ruby 3.1 `load` is safe by default. Why do we need `safe_load` then?

Reply:
> I'm not sure I got it. `safe_load` is not outdated, is it?
> Also, why do you need `# rubocop:disable Security/YAMLLoad`? `safe_load` doesn't have any security issues.

By outdated directive, I meant the rubocop directive 🙂 

> Also, [here](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Security/YAMLLoad) they say that in Ruby 3.1 `load` is safe by default. Why do we need `safe_load` then?

Aaah.. I didn't see that `load` was now an alias for `safe_load`. I'll refactor.